server/client.go

```diff
@@ -252,6 +255,18 @@ type AutoCompleteResult struct {
 	Results []Result `json:"results"`
 }
 
+type Item struct {
+	Name string `json:"name"`
+}
+
+type Group struct {
```

Comment:
Is this a UserGroup?

Reply:
or VisibilityUserGroup? Is this group being used for anything else?

Reply:
Now I have changed this to `GroupsInfo`.It indicate all the groups to which a commentator belongs to.

Reply:
How about `JiraUserGroupCollection` for the collection and `JiraUserGroup` for the singular group?